app/src/main/java/com/onboarding/pronosticAR/utils/WindDirectionConverter.kt

```diff
@@ -0,0 +1,14 @@
+package com.onboarding.pronosticAR.utils
+
+object WindDirectionConverter {
+    fun getWindDirection(wind: Int): String {
+        if (wind in 21..69) return ConstantUtils.NORTHEAST
+        if (wind in 70..110) return ConstantUtils.EAST
+        if (wind in 111..159) return ConstantUtils.SOUTHEAST
+        if (wind in 160..200) return ConstantUtils.SOUTH
+        if (wind in 201..249) return ConstantUtils.SOUTHWEST
+        if (wind in 250..290) return ConstantUtils.WEST
+        if (wind in 291..339) return ConstantUtils.NORTHWEST
+        return ConstantUtils.NORTH
```

Comment:
Facu, it's full of hardcodes , put on constants pls

Reply:
wops, forgot about those

Reply:
done